samples/attested_tls/client/enc/Makefile

```diff
@@ -10,6 +10,20 @@ INCDIR=$(shell pkg-config oeenclave-$(C_COMPILER) --variable=includedir)
 
 .PHONY: all build clean run
 
+SRC_FILES = ecalls.cpp mbedtls_client.cpp cert_verify_config.cpp ../../common/cert_verifier.cpp ../../common/identity_verifier.cpp ../../common/utility.cpp ../../common/mbedtls_utility.cpp
+OBJ_FILES = ecalls.o mbedtls_client.o cert_verify_config.o cert_verifier.o identity_verifier.o utility.o mbedtls_utility.o tls_client_t.o
+ADDITIONAL_LDFLAGS = -lmbedtls -lmbedcrypto
+LDFLAGS_CUSTOM = $(LDFLAGS)
+
+ifeq (${OE_CRYPTO_LIB}, openssl)
+	SRC_FILES = ecalls.cpp openssl_client.cpp cert_verify_config.cpp ../../common/verify_callback.cpp ../../common/utility.cpp ../../common/openssl_utility.cpp
+	OBJ_FILES = ecalls.o openssl_client.o cert_verify_config.o verify_callback.o utility.o openssl_utility.o tls_client_t.o
+	ADDITIONAL_LDFLAGS = -lopensslssl -lopensslcrypto
+	LDFLAGS_CUSTOM = $(subst -loecryptombedtls, -loecryptoopenssl, $(LDFLAGS))
+	# By default openenclave include mbedtls crypto library through package configuration files
```

Comment:
We'd discussed some days ago about removing the default inclusion of mbedtls. Otherwise there is inconsistency in that there is no default crypto library in cmake based build where as mbedtls is the default library in Makefile based build.
Are we planning to address that as part of this PR?

Reply:
I want to address that as a separate PR but as part of release V0.13.  Just keeping it as a separate PR as it involves changes in other sample application make files too.

Reply:
In this case, should we have the pkgconfig PR come before this PR?

Reply:
Yes created PR https://github.com/openenclave/openenclave/pull/3756